policies_sms/SMSTemplates/DefaultSMSTemplates.py

```diff
@@ -0,0 +1,127 @@
+from django.utils.translation import ugettext as _
+
+
+class DefaultSMSTemplates:
+
+    def get_all(self):
+        output = {}
+        for attr in self.__dir__():
+            if attr.startswith('__') or type(getattr(self, attr, None)) != str:
+                continue
+            else:
+                output[attr] = getattr(self, attr, None)
+
+        return output
+
+    @property
+    def sms_on_activation(self):
+        return _(
+            "The policy for the family/group "
+            "%(InsuranceID)s - %(Name)s with the insurance product "
+            "%(ProductCode)s -   %(ProductName)s  has been activated from the date "
+            "%(EffectiveDate)s until the expiry date  %(ExpiryDate)s. "
+        )
+
+    @property
+    def sms_on_effective(self):
+        return _(
+            "The policy for the family/group %(InsuranceID)s - "
+            "%(Name)s with the insurance product %(ProductCode)s -   "
```

Comment:
Make sure to remove unnecessary space chars. Here at the end.

Reply:
Spaces removed